OpenSim/Actuators/Test/testActuators.cpp

```diff
@@ -322,7 +322,7 @@ void testTorqueActuator()
 
     model->computeStateVariableDerivatives(state);
 
-    double fKE = model->getMatterSubsystem().calcKineticEnergy(state);
+    /*double fKE = */model->getMatterSubsystem().calcKineticEnergy(state);
```

Comment:
@aseth1 
This variable is currently unused. Maybe the author forgot to use it to test. Feel free to correct it by adding the appropriate test if you like. If you think it should be another PR, :+1:.